src/methods/methodMapping.ts

```diff
@@ -0,0 +1,35 @@
+import { Close, Connect, SendData } from '../__GENERATED_TYPES__';
+import IframeExecutionEnvironmentTransport from '../transports/IframeExecutionEnvironmentTransport';
+
+export interface IMethodMapping {
```

Comment:
Nit: We don't tend to use the "I" prefix convention elsewhere :thinking: We had done it in the first version of OpenRPC, but I thought Erik was trying to move away from that.

Some people like the prefix, but I don't think it's as necessary in TypeScript as in other languages, because the way interfaces are used makes it obvious what they are.